sdk/core/Azure.Core/src/Pipeline/HttpPipelineMessage.cs

```diff
@@ -1,42 +1,23 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using Azure.Core.Collections;
-using System;
-using System.ComponentModel;
 using System.Threading;
 
 namespace Azure.Core.Pipeline
 {
-    public partial class HttpPipelineMessage  : IDisposable
+    public struct HttpPipelineMessage
     {
-        internal OptionsStore _options = new OptionsStore();
```

Comment:
Is anybody using the OptionsStore type now?

Reply:
Nope.

Reply:
Should we remove it too?

Reply:
Just did